Sources/Units/Unit/Fraction.swift

```diff
@@ -0,0 +1,163 @@
+
+
+public struct Fraction: Hashable, Equatable, Sendable {
```

Comment:
Could we provide documentation about this type? Specific areas that would be helpful to document after reading through:

- `init` automatically reduces the input fraction
- String representation uses `|`

Reply:
Yes. Will do. I'll make a doc pass for everything I've added.